app/controllers/students/status_updates_controller.rb

```diff
@@ -55,4 +53,12 @@ def status_update_params
     params.require(:activity).permit(:title, :content)
   end
 
-end
+
+  private
+
+  def status_updates
+    #order = DESC; set in model
+    @status_updates = current_team.status_updates.ordered
```

Comment:
This could do with a `||=` assignment (memoization). That way, subsequent calls to `status_updates` won't trigger another database request. It's only called once in the view so it doesn't matter with the current implementation. But since it's being exposed as a helper method, it's better to not care how or how often a view template will call this.